src/lib/profiles/device-description/DeviceDescription.h

```diff
@@ -125,7 +125,7 @@ class NL_DLL_EXPORT WeaveDeviceDescriptor
         kMaxSerialNumberLength                  = 32,  /**< Maximum serial number length. */
         kMaxPairingCodeLength                   = 16,  /**< Maximum pairing code length. */
         kMaxRendezvousWiFiESSID                 = 32,  /**< Maximum WiFi ESSID for Rendezvous length. */
-        kMaxSoftwareVersionLength               = 32   /**< Maximum software version length. */
+        kMaxSoftwareVersionLength               = 128   /**< Maximum software version length. */
```

Comment:
This will blow up the size on small devices unnecessarily.  Can't we make this a configurable option with a default of 32?

Reply:
Agreed. Will post an update.